spec/features/users/new_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe 'As a Visitor' do 
+    describe 'When I visit the welcome page' do 
+        it 'I can register as a new user' do
+
+            visit '/'
+
+            expect(page).to have_content("Register")
+
+            click_on "Register"
```

Comment:
Did you not run in to any issues here with finding/clicking the new button? I had some problems(which is why I had to next it so deep) in my own visitor tests to make sure the register button was actually hit correctly.

Reply:
No I didn't have a problem at all. other than I had to add more info to the registration